clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -1272,50 +1272,49 @@ String coordinatorKey() {
         public void handleResponse(AbstractResponse response) {
             TxnOffsetCommitResponse txnOffsetCommitResponse = (TxnOffsetCommitResponse) response;
             boolean coordinatorReloaded = false;
-            boolean hadFailure = false;
             Map<TopicPartition, Errors> errors = txnOffsetCommitResponse.errors();
 
+            log.debug("Received TxnOffsetCommit response for consumer group {}: {}", builder.consumerGroupId(),
+                    errors);
```

Comment:
Do you think it would make sense to put an `if (!errors.isEmpty())` check around this log line for the cases when there are no errors? If I see it correctly it'd print `Received TxnOffsetCommit response for consumer group my-consumer-group:` which is I think is a bit misleading, and also I suppose it would be empty most of the times (meaning no errors).

Reply:
Ok, I think I see now, thanks for the explanation.